boards/b-l072z-lrwan1/Makefile.include

```diff
@@ -12,8 +12,5 @@ include $(RIOTMAKE)/tools/serial.inc.mk
 # this board has an on-board ST-link adapter
 export DEBUG_ADAPTER ?= stlink
 
-# call a 'reset halt' command before starting the debugger
```

Comment:
Please do not call this commit "revert" as it is not reverting the change, only using the now common definition.